src/Abp.ZeroCore/Authorization/Roles/AbpRoleStore.cs

```diff
@@ -49,15 +50,21 @@ public class AbpRoleStore<TRole, TUser> :
         private readonly IRepository<TRole> _roleRepository;
         private readonly IUnitOfWorkManager _unitOfWorkManager;
         private readonly IRepository<RolePermissionSetting, long> _rolePermissionSettingRepository;
+        private readonly IRepository<RoleClaim, long> _roleClaimRepository;
+        private readonly IAsyncQueryableExecuter _asyncQueryableExecuter;
 
         public AbpRoleStore(
             IUnitOfWorkManager unitOfWorkManager,
             IRepository<TRole> roleRepository, 
-            IRepository<RolePermissionSetting, long> rolePermissionSettingRepository)
+            IRepository<RolePermissionSetting, long> rolePermissionSettingRepository, 
+            IRepository<RoleClaim, long> roleClaimRepository, 
+            IAsyncQueryableExecuter asyncQueryableExecuter)
         {
```

Comment:
I think the usual practice for async queryable executor is public property injection

Reply:
Thanks @ryancyq It has changed.